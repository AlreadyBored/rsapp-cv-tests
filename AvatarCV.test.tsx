import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import AvatarCV from '../AvatarCV';

describe('AvatarCV', () => {
  describe('Should render correctly', () => {
    it('with source', () => {
      const src = 'https://example.com';

      const output = shallow(<AvatarCV src={src} />);
      expect(shallowToJson(output)).toMatchSnapshot();
    });

    it('without source', () => {
      const output = shallow(<AvatarCV src={null} />);
      expect(shallowToJson(output)).toMatchSnapshot();
    });
  });
});
