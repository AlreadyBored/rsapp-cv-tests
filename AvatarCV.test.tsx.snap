// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AvatarCV Should render correctly with source 1`] = `
<Avatar
  icon={null}
  shape="circle"
  size={120}
  src="https://example.com"
/>
`;

exports[`AvatarCV Should render correctly without source 1`] = `
<Avatar
  icon={<ForwardRef(UserOutlined) />}
  shape="circle"
  size={120}
/>
`;
